package.json

```diff
@@ -72,15 +72,15 @@
     "build:cjs": "cross-env BABEL_ENV=cjs babel source --out-dir src",
     "build:umd": "cross-env NODE_ENV=development rollup -c -o dist/ramda.js",
     "build:umd:min": "cross-env NODE_ENV=production rollup -c -o dist/ramda.min.js",
-    "build": "npm run build:es && npm run build:cjs && npm run build:umd && npm run build:umd:min",
+    "build": "npm-run-all --parallel build:**",
     "partial-build": "node ./scripts/partialBuild",
     "clean": "rimraf es/* src/* dist/* coverage/*",
-    "prepare": "npm run clean && npm run build",
+    "prepare": "npm-run-all --sequential clean build",
```

Comment:
While I understand the rationale for the change above, why do you think this is better than the `&&` version?  Do you just see greater consistency by always using `npm-run-all`, or is it that this version is more explicit?

Reply:
There's no advantage to use `npm-run-all --sequential` than `&&` version, just looks consistent.
I have changed `npm-run-all --sequential` back to `&&`, and make `test` run `spec` and `lint` parallel.